README.md

```diff
@@ -1,2 +1,53 @@
-# coc-camel
-Extension for Vim coc to provide Camel Textual Edition support
+# Camel LS for Vim and Neovim  
+
+This is short instructions for how to integrate Camel LS with Vim editor using coc.nvim (https://github.com/camel-tooling/camel-language-server).
+
+# Text Editing capabilities of Camel XML using coc.nvim
+
+![xmlnvim](images/xmlnvim.gif)
+
+# Text Editing capabilities of Camel URI with Camel JAVA using coc.nvim
+
+![javanvim](images/javanvim.gif)
+
+## Prerequisites
+
+Add these plugins in your `.vimrc` or `init.vim`
+```
+Plug 'neoclide/coc.nvim', {'branch': 'release'}
+```
+
+# Install Camel LS to Vim and Neovim using Neoclide coc.nvim LSP Client 
+
+### In a `.vimrc` file:
+```
+" Initialize plugin system
+  "
+  call plug#begin()
+  
+  " Shorthand notation; fetches https://github.com/neoclide/coc.nvim
+  Plug 'neoclide/coc.nvim', {'branch': 'release'}
+  
+  call plug#end()
+```
+### Configuration is required to make coc.nvim easier to work with. This is done as much as possible to avoid conflict with other plugins.
```

Comment:
I guess this is not required. if you want you might provide a link to the documentation of coc.nvim about this subject